src/wp-includes/script-loader.php

```diff
@@ -214,6 +214,39 @@ function wp_get_script_polyfill( $scripts, $tests ) {
 	return $polyfill;
 }
 
+/**
+ * Registers development scripts that integrate with `@wordpress/scripts`.
+ *
+ * @see https://github.com/WordPress/gutenberg/tree/trunk/packages/scripts#start
+ *
+ * @since 6.0.0
+ *
+ * @param WP_Scripts $scripts WP_Scripts object.
+ */
+function wp_register_development_scripts( $scripts ) {
+	if ( ! defined( 'SCRIPT_DEBUG' ) || ! SCRIPT_DEBUG ) {
+		return;
+	}
+
+	$development_scripts = array(
+		'react-refresh-entry',
+		'react-refresh-runtime',
+	);
+
+	foreach ( $development_scripts as $script_name ) {
+		$assets = include ABSPATH . WPINC . "/js/dist/development/$script_name.php";
```

Comment:
Please consider this a nitpick, but for consistency, I'd either use the concatenation operator or curly braces to concatenate strings across this function.

```php
$assets = include ABSPATH . WPINC . '/js/dist/development/' . $script_name . '.php';
```
or
```php
$assets = include ABSPATH . WPINC . "/js/dist/development/${script_name}.php";
```
Performance-wise, the first option is better.
It's a bit hard to read:
```php
"/wp-includes/js/dist/development/$script_name.js",
```